demos/blockfactory/block_library_storage.js

```diff
@@ -60,17 +57,15 @@ BlockLibraryStorage = function(blockLibraryName, opt_blocks) {
  * Reads the named block library from local storage and saves it in this.blocks.
  */
 BlockLibraryStorage.prototype.loadFromLocalStorage = function() {
-  // goog.global is synonymous to window, and allows for flexibility
-  // between browsers.
-  var object = goog.global.localStorage[this.name];
+  var object = localStorage[this.name];
```

Comment:
none of the browsers require this to be window.?

Reply:
window is the global scope in a browser context: https://stackoverflow.com/a/12660102/152543